platform/api/aws/images.go

```diff
@@ -349,7 +349,7 @@ func (a *API) deregisterImageIfExists(name string) error {
 }
 
 // Remove all uploaded data associated with an AMI, also in other given regions.
-func (a *API) RemoveImage(name, s3BucketName, s3ObjectPath string, otherRegions []string) error {
+func (a *API) RemoveImage(name, imageName, s3BucketName, s3ObjectPath string, otherRegions []string) error {
```

Comment:
It would be good to rename the `name` to `amiName` to make it more verbose.

Reply:
Yes, done